src/Bicep.Cli/Services/Evaluation.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Azure.Deployments.Core.Definitions.Schema;
+
+namespace Bicep.Cli.Services
+{
+    public class Evaluation
+    {
+        public Template? Template { get; }
+        public InvalidOperationException? Error { get; }
+        public Dictionary<string, bool> Assertions { get; }
```

Comment:
I'd recommend renaming `Assertions` -> `AllAssertions` to make it clearer that this is the full list (rather than just the successful ones)